src/main/java/ui/issuepanel/PanelControl.java

```diff
@@ -105,6 +113,7 @@ public FilterPanel addPanelAt(int index) {
 
         updatePanelIndices();
         setCurrentlySelectedPanel(Optional.of(index));
+        mainStage.show();
```

Comment:
Is this still needed? If not, `PanelControl` no longer needs to know about the stage.